lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -181,9 +181,10 @@ def self.replace_in(previous_text, new_text, path, allow_empty: false)
         if new_text.to_s.strip.empty? && !allow_empty
           UI.user_error!("Missing `new_text` in call to `replace_in`, looking for replacement for #{previous_text} 😵.")
         end
-        sed_regex = "s|#{previous_text.sub('.', '\\.')}|#{new_text}|"
-        backup_extension = '.bck'
-        Actions.sh("sed", "-i#{backup_extension}", sed_regex, path)
+        original_text = File.read(path)
+        replaced_text = original_text.gsub(previous_text, new_text)
+        File.write("#{path}.bck", original_text)
```

Comment:
Honestly, I'm not sure if we need the backup file having github... But I thought I would keep the same functionality we had with `sed` just in case

Reply:
let's remove the `.bck`, it's never been particularly helpful